### DISCUSSION TITLE
Lesson4: simple question about price converter

### DISCUSSION THREAD

**Discussion by NanaChiki** [Cyfrin/foundry-full-course-cu#191] [Q&A] [answered]
Why I sent 0.02ETH in terms of USD into the FundMe contract is greater than MIMIMUM_USD value of 5e18? 
I'm still confusing about a reason why FundMe contract was be able to receive 0.02ETH that might be less amount of minimum USD value.

**Comment by ghost** [comment]
@NanaChiki that depends on the price of an ether. If `20000000000000000 wei * ethPriceInWei >= 5e18` then the condition is satisfied.

0.02 ETH in wei is 20000000000000000.

**Reply by NanaChiki** [reply]
I'm grateful for your help!

**Comment by thurendous** [comment, marked as answer]
It depends on the price of the oracle itself. 
Say the price of ETH is $2000, then 0.02ETH * $2000 = $40. In you case, if 0.05ETH is less than $5, then the price should be less than this: 0.02ETH * $250 = $5. 
There are 2 options for you:  
1. You can raise the price of the mock contract's price
2.  you can send more ETH to cover it.

**Reply by NanaChiki** [reply]
I totally understand this calculation.
Thanks for taking the time!!
